src/docs/product/integrations/integration-platform/webhooks.mdx

```diff
@@ -73,33 +73,34 @@ if digest != expected_digest:
 
 All webhook requests have some common elements.
```

Comment:
I just noticed that we describe our requests as having this common structure that we don't always maintain! This goes along with my suggestion that we enforce some class/function responsible for keeping the payloads identical. Cool to see that it was originally designed like this, even if we lost the plot a bit.

I don't think you need you need to change this at all, I just wanted to mention that we claim it in our docs, so we should probably get on enforcing it through code.